### ISSUE TITLE
Incorrect output from CTC decoder

### ISSUE THREAD

**Issue by dreamibor** [PaddlePaddle/DeepSpeech#245]
Hi,

What's the input shape of the CTC decoder, is it (time_setps, num_classes)? The output from the CTC decoder is like "(nan, "ud rkn z ob elm c bi c vb wpx usr bg wr tok r weu ktt ty f ' pk ej stj w qe pd ayu gbs r jnl gc h bli po ggc h w j roi qk s e ech esf b wnm siba qa fu")" while it will be okay to use greedy decoder. I am using Python 3 and compile the CTC decoder for Python 3.5. Is that relevant?

Thanks in advance!

**Comment by dreamibor** [comment]
Haha, it turns out I have to put a softmax layer between the logits and the CTC decoder to normalise the input.

**Comment by sruteesh-pivot** [comment]
Hi @dreamibor Were u able to run deepspeech on python3.5?
